react-frontend/src/index.css

```diff
@@ -0,0 +1,10 @@
+body {
+  margin: 0;
+  background-color: #fafafa;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
@jainpawan21 This font won't load.
Import this font. Check [this](https://fonts.google.com/specimen/Montserrat?selection.family=Montserrat).

Reply:
These fonts are loading perfectly and working fine!
Should I download local fonts?

Reply:
No, can't we import?

Reply:
I forgot to copy that link. Fixed Now :)